gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Added username to header

### Description

Added username next to logout in header.

### Testing Directions

```
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
python manage.py runserver
```

Make a user and navigate through application to see the username next to logout.

### Checklist
